CodeSparks/Areas/Identity/Pages/Account/Manage/Index.cshtml

```diff
@@ -1,35 +1,104 @@
 ﻿@page
 @model IndexModel
+@inject ISocialNetworkService SocialNetworkService
+@* @model AppUser *@
+
 @{
     ViewData["Title"] = "Profile";
     ViewData["ActivePage"] = ManageNavPages.Index;
 }
 
-<h3>@ViewData["Title"]</h3>
-<partial name="_StatusMessage" for="StatusMessage" />
-<div class="row">
-    <div class="col-md-6">
-        <form id="profile-form" method="post">
-            <div asp-validation-summary="ModelOnly" class="text-danger" role="alert"></div>
-            <div class="form-floating mb-3">
-                <input asp-for="Username" class="form-control" placeholder="Please choose your username." disabled />
-                <label asp-for="Username" class="form-label"></label>
+<head>
```

Comment:
This page uses layout, defined in _ViewStart.cshtml. And in that layout we already have <head> element. So this will create another one, which is not the best way for html.
If you need to add more styles - you can add them to layout (if they are used for all pages), or add them to a section (in this project it is - "HeadStyles").